packages/typespec-test/test/nestedClient/generated/typespec-ts/src/storageClient.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { BlobClient, BlobClientOptionalParams } from "./blob/blobClient.js";
+import {
+  createStorage,
+  StorageContext,
+  StorageClientOptionalParams,
+} from "./api/index.js";
+import { DownloadOptionalParams } from "./api/options.js";
+import { download } from "./api/operations.js";
+import { Pipeline } from "@azure/core-rest-pipeline";
+
+export { StorageClientOptionalParams } from "./api/storageContext.js";
+
+export class StorageClient {
+  private _client: StorageContext;
+  /** The pipeline used by this client to make requests */
+  public readonly pipeline: Pipeline;
+  /** The parent client parameters that are used in the constructors. */
+  private _parentClientParams: {
```

Comment:
nit: I feel like it should not be `parent` but just `_ClientParams`? If we named it parent it usually stands for the `StorageClient`'s parent parameters.